src/Compilers/Core/Portable/Operations/ChildOperationList.cs

```diff
@@ -65,13 +62,13 @@ public ImmutableArray<IOperation> ToImmutableArray()
         /// <see cref="Enumerator.MoveNext"/> has returned false will throw an <see cref="InvalidOperationException"/>.
         /// </summary>
         [NonDefaultable]
-        internal struct Enumerator : IEnumerator<IOperation>
+        public struct Enumerator : IEnumerator<IOperation>
         {
             private readonly Operation _operation;
             private int _currentSlot;
             private int _currentIndex;
 
-            public Enumerator(Operation operation)
+            internal Enumerator(Operation operation)
```

Comment:
based on the impl here, it appears as if Operations do understand slots/indexes.  so i think we should expose ChildOepration list and add more to it later on demand.  for example, things like First()/Last() and other helpers on ChlidSyntaxList have proven *very* helpful.  Thank you for coming to my Ted talk.